faq.md

```diff
@@ -0,0 +1,160 @@
+
+#### Is Operaton truly open?
+Yes! All discussions and decisions are public. If you have questions, we encourage you to ask them in the [Forum](https://forum.operaton.org/) rather than in private.
+
+### Release Planning
+
+#### Why is Operaton still beta?
+Operaton is in beta because it is still *undergoing stabilization* and *compatibility testing*. The project team is working hard to ensure that the transition from Camunda 7 is as smooth as possible. At the same time, the project is *modernizing the codebase* to ensure long-term sustainability. Until these goals are met, the project will remain in beta.
+
+#### What is the release cycle for Operaton?
+While nothing is finalized yet, there is a high likelihood that releases will follow a fixed cadence, possibly half-yearly or quarterly, depending on community needs and available resources. The project team will provide more details as the project matures.
+
+#### Will there be an LTS (Long-Term Support) version?
+Yes, version 1.x will be considered LTS. However, Operaton itself does not provide SLAs (Service Level Agreements). Companies that offer Operaton support services will need to ensure long-term maintenance.
+
+#### What happens after version 1.x?
+After establishing a stable 1.x version, the project will *gradually modernize and evolve*. The goal is to *stay relevant for the next decade* while maintaining backward compatibility.
+
+### Security
+
+#### How secure is Operaton?
+Operaton follows security best practices, including authentication, authorization, and encrypted database connections.
+
+#### Can I use Operaton with Single Sign-On (SSO)?
+Yes, Operaton supports authentication via OAuth2, OpenID Connect, and LDAP.
+
+#### What security policies are in place for Operaton?
+Operaton follows a security policy that includes vulnerability management, responsible disclosure, and best practices for open-source security.
+
+#### Does Operaton provide SBOMs?
+Yes, Operaton provides *[Software Bill of Materials (SBOMs)](https://en.wikipedia.org/wiki/Software_supply_chain)* to help users understand the dependencies and security of the software they are using.
+
+### Migration from Camunda 7 CE
+
+#### Can I migrate from Camunda 7 Community Edition (CE) to Operaton?
+Yes! Operaton is designed to provide *smooth transition* for Camunda 7 CE. For the case that minor adjustments are required they will be documented extensively in our migration guide.
+
+#### Will my existing BPMN processes work in Operaton?
+Yes. The 1.x versions of Operaton prioritize *full compatibility with Camunda 7*. Your BPMN models, process definitions, and execution logic will continue to work without changes.
+
+#### How do I migrate from Camunda 7 to Operaton?
+The migration process is straightforward:
+1. Upgrade to Camunda 7 CE *version 7.24*, which is the latest release before its end of life.
```

Comment:
Implies that a migration is not possible before October 2025?

Reply:
Who would migrate before our 1.0 release?
When we have exacter dates we can extend the FAQ.